src/DotNet.Status.Web/.config/settings.json

```diff
@@ -15,18 +15,29 @@
     "InstrumentationKey": "[vault(app-insights-instrumentation-key)]"
   },
   "AzureDevOps": {
-    "BaseUrl": "https://dev.azure.com",
+    "build-monitor/dnceng": {
+      "Organization": "dnceng",
+      "AccessToken": "[vault(dn-bot-dnceng-build-r-code-r-project-r-profile-r)]",
+      "MaxParallelRequests": 10
+    },
+    "dnceng": {
+      "Organization": "dnceng",
+      "AccessToken": "[vault(dn-bot-dnceng-workitems-rw)]",
+      "MaxParallelRequests": 10
+    },
+    "dnceng-public": {
+      "Organization": "dnceng-public",
+      "AccessToken": "[vault(dn-bot-dnceng-workitems-rw)]",
```

Comment:
Is this the correct access token? It should only have workitems privileges, and we aren't doing work items in dnceng-public

Reply:
(And is also a dnceng token, not a dnceng-public token)

Reply:
This section isn't actually used. I'll remove it